src/game/game.ts

```diff
@@ -116,6 +117,8 @@ export class Game extends EventEmitter<GameEvents> {
   }
 
   @action handleInput(key: string) {
+    console.log()
```

Comment:
Oops, seems like you left a console.log here :)

Reply:
whoops! thanks for catching that